calyx-py/test/correctness/matrix.py

```diff
@@ -0,0 +1,71 @@
+import calyx.builder as cb
+
+def insert_matmul_component(prog):
```

Comment:
I want to eventually point to this .py file from the documentation, so could you please add a docstring that explains what this method does?

Reply:
Nope, looks good to me